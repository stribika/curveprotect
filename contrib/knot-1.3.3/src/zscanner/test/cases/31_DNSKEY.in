$ORIGIN	.
$TTL	1

; OK
@	DNSKEY	0	0	0	AA==		; The simplest variant
@	DNSKEY	65535	255	255	AA==		; Maximal numbers
@	DNSKEY	0	0	RSAMD5	AA==		; Algorithm mnemonic
@	DNSKEY	0	0	0	Zm8=		; One char padding
@	DNSKEY	0	0	0	Zm9v		; Without padding
@	DNSKEY	0	0	0	Zm9vYg==	; Two base64 blocks
@	DNSKEY	0	0	0	Zm9v YmE=	; Two base64 blocks with blank space between them
@	TYPE48	\# 5 0000000000				; TYPE + Hexadecimal rdata
@	TYPE48	0	0	0	AA==		; TYPE
@	dnskey	0	0	0	AA==		; Type in lower-case

; KO
@	DNSKEY
@	DNSKEY						; Empty rdata
@	DNSKEY	\# 0					; Hex empty rdata
@	DNSKEY	\#					; Missing hex length
@	DNSKEY	65536	0	0	AA==		; Type overflow
@	DNSKEY	0	256	0	AA==		; Key tag overflow
@	DNSKEY	0	0	256	AA==		; Algorithm overflow
@	DNSKEY	0	0	0	A		; Continuous block length must be multiple of 4
@	DNSKEY	0	0	0	AB		; Continuous block length must be multiple of 4
@	DNSKEY	0	0	0	ABC		; Continuous block length must be multiple of 4
@	DNSKEY	0	0	0	AA ==		; Continuous block length must be multiple of 4
@	DNSKEY	0	0	0	A===		; Bad padding
@	DNSKEY	0	0	0	=		; Bad padding
@	DNSKEY	0	0	0	==		; Bad padding
@	DNSKEY	0	0	0	===		; Bad padding
@	DNSKEY	0	0	0	====		; Bad padding
@	DNSKEY	0	0	0			; Missing item
@	DNSKEY	\# 5 000000000000			; Too long rdata
@	DNSKEY	\# 6 0000000000				; Bad rdata length
