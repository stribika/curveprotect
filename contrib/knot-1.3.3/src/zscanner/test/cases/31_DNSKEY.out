OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0030
RDATA=0000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0030
RDATA=FFFFFFFF00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0030
RDATA=0000000100
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0030
RDATA=00000000666F
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0030
RDATA=00000000666F6F
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0030
RDATA=00000000666F6F62
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0030
RDATA=00000000666F6F6261
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0030
RDATA=0000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0030
RDATA=0000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0030
RDATA=0000000000
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_ENUMBER16_OVERFLOW
------
WARNG=ZSCANNER_ENUMBER8_OVERFLOW
------
WARNG=ZSCANNER_ENUMBER8_OVERFLOW
------
WARNG=ZSCANNER_EBAD_BASE64_CHAR
------
WARNG=ZSCANNER_EBAD_BASE64_CHAR
------
WARNG=ZSCANNER_EBAD_BASE64_CHAR
------
WARNG=ZSCANNER_EBAD_BASE64_CHAR
------
WARNG=ZSCANNER_EBAD_BASE64_CHAR
------
WARNG=ZSCANNER_EBAD_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA_LENGTH
------
WARNG=ZSCANNER_EBAD_RDATA_LENGTH
------
